packages/ng/core/translate/intl.model.ts

```diff
@@ -16,3 +16,16 @@ export function getIntl<T>(translationsToken: InjectionToken<LuTranslation<T>> |
 
 	return translations['en'] ?? translations['en-GB'] ?? translations['en-US'];
 }
+
+export function intlInputOptions<T>(token: InjectionToken<LuTranslation<T>>): [T, InputOptionsWithTransform<T, Partial<T>>] {
```

Comment:
That's elegant but I needed a minute to understand what the transform was. It's not immediately clear that it's a component `input` option (even though it's right there in the name 😂)

Maybe a bit of a jsdoc here?

Reply:
Sure! This PR needs polish 😁